Source/Core/Core/State.cpp

```diff
@@ -391,6 +399,11 @@ static void CompressAndDumpState(CompressAndDumpState_args save_args)
 
 void SaveAs(const std::string& filename, bool wait)
 {
+	if (NetPlay::IsNetPlayRunning())
+	{
+		OSD::AddMessage("Access to savestates disabled in Netplay to prevent desyncs");
+		return;
+	}
```

Comment:
It does pause the core (on one of the clients, not all of them), so it might affect Netplay negatively. If it _really_ causes issues in practise; no idea.